support-frontend/assets/components/page/page.jsx

```diff
@@ -31,6 +31,15 @@ export default function Page(props: PropTypes) {
     </div>
   ) : null;
 
+  React.useEffect(() => {
```

Comment:
I seem to get an error in the build even though this line works, how can I fix it? googling didn't help much.
```
12:44:10
  12:44:10
  Cannot call `React.useEffect` because property `useEffect` is missing in object type [1].
12:44:10
  12:44:10
     assets/components/page/page.jsx:34:3
12:44:10
            v----------------------
12:44:10
      34|   React.useEffect(() => {
12:44:10
      35|     // setTimeout(() => {
12:44:10
      36|     if (window.location.hash) {
12:44:10
      37|       const hashElement = document.getElementById(window.location.hash.substr(1));
12:44:10
      38|       if (hashElement) { hashElement.scrollIntoView(); }
12:44:10
      39|     }
12:44:10
      40|     // }, 1);
12:44:10
      41|   }, []);
12:44:10
            -----^
12:44:10
  12:44:10
  References:
12:44:10
     /tmp/flow/flowlib_1dbc5f75/react.js:271:26
12:44:10
                                   v-
12:44:10
     271|   declare export default {|
12:44:10
     272|     +DOM: typeof DOM,
12:44:10
     273|     +PropTypes: typeof PropTypes,
12:44:10
     274|     +version: typeof version,
12:44:10
     275|     +checkPropTypes: typeof checkPropTypes,
12:44:10
     276|     +createClass: typeof createClass,
12:44:10
     277|     +createContext: typeof createContext,
12:44:10
     278|     +createElement: typeof createElement,
12:44:10
     279|     +cloneElement: typeof cloneElement,
12:44:10
     280|     +createFactory: typeof createFactory,
12:44:10
     281|     +createRef: typeof createRef,
12:44:10
     282|     +isValidElement: typeof isValidElement,
12:44:10
     283|     +Component: typeof Component,
12:44:10
     284|     +PureComponent: typeof PureComponent,
12:44:10
     285|     +Fragment: typeof Fragment,
12:44:10
     286|     +Children: typeof Children,
12:44:10
     287|   |};
12:44:10
            -^ [1]
```

Reply:
rupert has helped with this, add 
```
// $FlowIgnore - required for hooks
import React, { useEffect } from 'react';
```